### PR TITLE
adding project_documentation, preface and fallback content

### PR DESCRIPTION
**Pull request by marcosgopen**
these adoc files are needed to navigate the documentation

### CODE REVIEW

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+To insert a special character into a `gedit` file, choose menu:Applications[Accessories > Character Map] from the main menu bar.
+Next, choose menu:Search[Find] from the `Character Map` menu bar, type the name of the character in the [label]#Search# field and click btn:[Next].
+The character you sought will be highlighted in the [label]#Character Table#.
+Double-click this highlighted character to place it in the [label]#Text to copy# field and then click the btn:[Copy] button.
+Now switch back to your document and choose menu:Edit[Paste] from the `gedit` menu bar. 
+____
+
+The above text includes application names; system-wide menu names and items; application-specific menu names; and buttons and text found within a GUI interface, all presented in proportional bold and all distinguishable by context. 
+
+``Mono-spaced Bold Italic`` or [app]```Proportional Bold Italic```
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
`Mono-spaced Bold Italic` or `Proportional Bold Italic`
```

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+Each class has its own associated set of permissions. 
+
+`Proportional Bold`
+
+This denotes words or phrases encountered on a system, including application names; dialog box text; labeled buttons; check-box and radio button labels; menu titles and sub-menu titles.
+For example: 
+
+____
+Choose menu:System[Preferences > Mouse] from the main menu bar to launch `Mouse Preferences`.
+In the [label]#Buttons# tab, click the [label]#Left-handed mouse# check box and click btn:[Close] to switch the primary mouse button from the left to the right (making the mouse suitable for use in the left hand).
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
In the `Buttons tab`, click the `Left-handed mouse` check box and click `Close` to switch the primary mouse button from the left to the right (making the mouse suitable for use in the left hand). 
```

Comment (jmfinelli) [reply src/main/asciidoc/project/fallback_content/conventions.adoc]: Same for all other occurrences in the text

**Comment by marcosgopen** [comment]
thanks Manuel, I'll double check those occurrences

**Comment by marcosgopen** [comment]
Occurrences checked and corrected

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+Whether mono-spaced bold or proportional bold, the addition of italics indicates replaceable or variable text.
+Italics denotes text you do not input literally or displayed text that changes depending on circumstance.
+For example: 
+
+____
+To connect to a remote machine using ssh, type `ssh `username`@`domain.name`` at a shell prompt.
+If the remote machine is _example.com_ and your username on that machine is john, type ``ssh john@example.com``. 
+
+The `mount -o remount `file-system`` command remounts the named file system.
+For example, to remount the [path]_/home_ file system, the command is ``mount -o remount /home``.
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
For example, to remount the `_/home_ file system`, the command is `mount -o remount /home`. 
```

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+For example: 
+
+____
+To connect to a remote machine using ssh, type `ssh `username`@`domain.name`` at a shell prompt.
+If the remote machine is _example.com_ and your username on that machine is john, type ``ssh john@example.com``. 
+
+The `mount -o remount `file-system`` command remounts the named file system.
+For example, to remount the [path]_/home_ file system, the command is ``mount -o remount /home``. 
+
+To see the version of a currently installed package, use the `rpm -q `package`` command.
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
To see the version of a currently installed package, use the `rpm -q package` command.
```

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+`Mono-spaced Bold Italic` or `Proportional Bold Italic`
+
+Whether mono-spaced bold or proportional bold, the addition of italics indicates replaceable or variable text.
+Italics denotes text you do not input literally or displayed text that changes depending on circumstance.
+For example: 
+
+____
+To connect to a remote machine using ssh, type `ssh `username`@`domain.name`` at a shell prompt.
+If the remote machine is _example.com_ and your username on that machine is john, type ``ssh john@example.com``.
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
If the remote machine is _example.com_ and your username on that machine is john, type `ssh john@example.com`.
```

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+The above text includes application names; system-wide menu names and items; application-specific menu names; and buttons and text found within a GUI interface, all presented in proportional bold and all distinguishable by context. 
+
+`Mono-spaced Bold Italic` or `Proportional Bold Italic`
+
+Whether mono-spaced bold or proportional bold, the addition of italics indicates replaceable or variable text.
+Italics denotes text you do not input literally or displayed text that changes depending on circumstance.
+For example: 
+
+____
+To connect to a remote machine using ssh, type `ssh `username`@`domain.name`` at a shell prompt.
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
To connect to a remote machine using ssh, type `ssh username@domain.name` at a shell prompt.
```

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+If the remote machine is _example.com_ and your username on that machine is john, type `ssh john@example.com`.
+
+The `mount -o remount `file-system`` command remounts the named file system.
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
The `mount -o remount file-system` command remounts the named file system.
```

File: src/main/asciidoc/project/fallback_content/conventions.adoc [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+For example, to remount the `_/home_ file system`, the command is `mount -o remount /home`. 
+
+To see the version of a currently installed package, use the `rpm -q package` command.
+It will return a result as follows: `package-version-release`. 
+____
+
+Note the words in bold italics above &mdash;username, domain.name, file-system, package, version and release.
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/conventions.adoc]:
```suggestion
Note the words in bold italics above -username, domain.name, file-system, package, version and release.
```

File: src/main/asciidoc/project/fallback_content/feedback.adoc [hunk 9]
```diff
@@ -0,0 +1,11 @@
+
+= We Need Feedback!
+(((feedback,contact information for this manual)))
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/feedback.adoc]: This is not needed I think

File: src/main/asciidoc/project/fallback_content/feedback.adoc [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+
+Please feel free to raise any issues you find with this document in our https://issues.jboss.org/browse/JBTM[issue tracking system]
+
+ifdef::backend-docbook[]
+[index]
+== Index
+// Generated automatically by the DocBook toolchain.
+endif::backend-docbook[]
```
Comment (jmfinelli) [inline src/main/asciidoc/project/fallback_content/feedback.adoc]: Is this needed?

Comment (marcosgopen) [reply src/main/asciidoc/project/fallback_content/feedback.adoc]: probably not, I will remove it.

File: src/main/asciidoc/project/fallback_content/feedback.adoc (hunk 9 above)
Comment (marcosgopen) [reply src/main/asciidoc/project/fallback_content/feedback.adoc]: I agree, thanks

**Review by jmfinelli — APPROVED** [review]

**Pull request merged**